src/main/kotlin/io/sdkman/broker/health/MongoHealthCheck.kt

```diff
@@ -0,0 +1,25 @@
+package io.sdkman.broker.health
+
+import io.sdkman.broker.app.AppRepo
+import ratpack.exec.Promise
+import ratpack.health.HealthCheck
+import ratpack.health.HealthCheck.Result
+import ratpack.registry.Registry
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class MongoHealthCheck @Inject constructor(private val appRepo: AppRepo) : HealthCheck {
+
+    private val unhealthyMessage = "Nothing found at application/alive in database."
```

Comment:
Same as above. I think if declared const `unhealthyMessage` gets inlined by the compiler.

Reply:
I don't feel so strongly about this, as in all these cases the entire class is a `@Singleton`, but I've changed all these fields to `const` now.